contracts/Ingress.sol

```diff
@@ -42,15 +42,15 @@ contract Ingress {
                 owner: msg.sender,
                 contractAddress: addr
             });
+
+        // Update registry indexing
+        contractKeys.push(name);
        } else {
             info.contractAddress = addr;
        }
         // update record in the registry
```

Comment:
Should this be conditional as well?   If the record exists, it's going to reassign it to the same value?

Reply:
I think that's what we want though?